src/Psalm/Checker/Statements/Expression/CallChecker.php

```diff
@@ -318,11 +318,11 @@ public static function analyzeFunctionCall(
                     $stmt->args,
                     $method_id,
                     $function_params,
+                    $code_location,
```

Comment:
Instead of this, could you just put the type into the docblock? It's designed to be a nullable type, not an optional one. It was bad of me to abuse the syntax, but :man_shrugging: